Makefile

```diff
@@ -0,0 +1,33 @@
+container_backend=podman
```

Comment:
should probably default to docker since more ppl seem to use that.

also add some note about how this doesnt work as expected with rootless docker and network == host